test-config/sensirion_test_setup.cpp

```diff
@@ -0,0 +1,12 @@
+#include "CppUTest/CommandLineTestRunner.h"
+
+#include "sensirion_i2c.h"
+
+int16_t i2c_reset() {
```

Comment:
This function shouldn't be needed, since it is part of `sensirion_common` named `sensirion_i2c_general_call_reset`. Maybe mark it as deprecated and just call `sensirion_i2c_general_call_reset` from it.

Reply:
Did that and tested it :) Removing the function can be done when I update the makefiles to distinguish between unit and hw tests , then I need to go through the tests anyway.